honeybee_schema/model.py

```diff
@@ -366,6 +374,40 @@ class Model(NamedBaseModel):
         description='The clockwise north direction in degrees.'
     )
 
+    units_system: UnitsSystems = Field(
```

Comment:
What about dropping `system` and use `units` instead?

Reply:
I was a little worried about it getting confused with all of the other things that have units in our libraries/schema but maybe this was over-cautious.  Now that I think about it "Units System" is what we use to describe SI vs IP and doesn't describe meters vs millimeters or feet vs inches. Ok, I'll change it to `units`.

Reply:
Done